symphony/lib/boot/func.utilities.php

```diff
@@ -8,32 +8,32 @@
  * Redirects the browser to a specified location. Safer than using a
  * direct header() call
  *
- *  @param string $url
+ *  @param string $location
+ *  @param string $status (optional)
```

Comment:
I guess it's ok to keep it simple. Developers using this should know what they're doing (and will notice pretty fast if something went wrong).